enhancements/installer/openstack-baremetal-compute.md

```diff
@@ -0,0 +1,193 @@
+---
+title: openstack-baremetal-compute
+authors:
+  - "@pierreprinetti"
+reviewers:
+  - "@openshift/team-openstack-cloud-provider"
+approvers:
+  - TBD
+creation-date: 2019-12-10
+last-updated: 2019-12-10
+status: provisional
+
+---
+
+# Openstack Baremetal Compute
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenStack's [Nova][openstack-nova] service can be configured to provision
+baremetal machines as OpenShift Compute Nodes.
+
+This enhancement will allow [Ironic][openstack-ironic]-provisioned machines to
+be used as OpenShift Compute Nodes, when they are made available as Nova
+flavors.
```

Comment:
So, from an operational standpoint, it makes a lot of sense to plug through the compute layer of OpenStack because ultimately, it should not matter. In fact, having the OpenStack cloud provider should allow selection of baremetal or virtual machines and operationally there shouldn't really be any difference between the two from the cloud provider perspective.

The OpenShift's baremetal provider also provides the same functionality pass-through direct to ironic, you just loose knowledge of a remote ironic deployment and the infrastructure because your handing the request over to an IaaS platform.

Reply:
nit. Nova Service -> OpenStack Compute service. 
I know is usually called nova but it is officially a compute service, it just happens that the default one installed is the Nova project.